### DISCUSSION TITLE
tutorials/intro-typescript/04-apollo-server

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#99] [course feedback]
# tutorials/intro-typescript/04-apollo-server

https://www.apollographql.com/tutorials/intro-typescript/04-apollo-server

**Comment by brian-yao** [comment]
Getting error: Error: Can't add plugins after the server has started after running app

**Reply by mabuyo** [reply]
Hey @brian-yao ! At what step are you getting this error? If it's after running the `npm install` for the packages, could you try stopping the app before doing so?

**Reply by brian-yao** [reply]
Hi @mabuyo I re-cloned the initial repo and found the issue has been resolved. Thanks.

**Comment by ChrisGrigg** [comment]
Great explanations, the most clear I've seen.
